examples/layer-shell.c

```diff
@@ -119,18 +144,113 @@ struct zwlr_layer_surface_v1_listener layer_surface_listener = {
 	.closed = layer_surface_closed,
 };
 
+static void wl_pointer_enter(void *data, struct wl_pointer *wl_pointer,
+		uint32_t serial, struct wl_surface *surface,
+		wl_fixed_t surface_x, wl_fixed_t surface_y) {
+	wl_surface_attach(cursor_surface,
+			wl_cursor_image_get_buffer(cursor_image), 0, 0);
```

Comment:
do you need `wl_surface_damage()` here?

Reply:
Not by my tests, since the surface never changes it's not important